.travis.yml

```diff
@@ -1,6 +1,51 @@
 language: csharp
 
-sudo: false  # use the new container-based Travis infrastructure 
+#dotnet cli require Ubuntu 14.04
+sudo: required
+dist: trusty
+
+#dotnet cli require OSX 10.10
+osx_image: xcode7.1
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+    - zlib1g
+
+os:
+  - osx
+
+env:
+  - CLI_VERSION="1.0.0.001494"
+
+before_install:
+  # Download script to install dotnet cli
+  - curl -L --create-dirs https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/install.sh -o ./scripts/obtain/install.sh
+  - find ./scripts -name "*.sh" -exec chmod +x {} \;
+  # use bash to workaround bug https://github.com/dotnet/cli/issues/1725
+  - sudo bash ./scripts/obtain/install.sh --channel "beta" --version "$CLI_VERSION"
+  - which dotnet
 
 script: 
   - ./build.sh All
+  # Build .NET Core with .NET CLI
+  # Workaround fsc doesnt work
+  - if test "$TRAVIS_OS_NAME" == "osx"; then sudo curl -L -s https://github.com/enricosada/fsharp-dotnet-cli-samples/raw/master/workarounds/1/osx.10.10-x64.zip | tar xvf - -C /usr/local/share/dotnet/cli/bin/ ; fi
+  # Workaround "Too many open files"
+  - ulimit -n 1024
+  # cli 
+  - dotnet restore
+  # Build Chessie
```

Comment:
Install of .NET CLI should be environment specific, it's ok to have it in the CI server file.

About build and test with .NET CLI, I would put this part into the build script, and also as part of this PR. Why do we need an other PR for moving things in the right places, when it's already clear now?

> if possibile i dont want to touch your build script because i dont know enough of it.

Provide two targets in the build script, one for Build and one for Test. There is nothing special about the build script, it's a standard FAKE script from project scaffold.